src/componets/API/weather_query.tsx

```diff
@@ -0,0 +1,38 @@
+import React,{useState, useEffect, FunctionComponent} from "react";
+import axios from "axios";
+
+
+export function WeatherDataGet(){
+    const [data1, setdata1] = useState<Array<string>>([]);
+    useEffect(() =>{
+        async function fetchData(){
+            let fetchUrl ='https://ktc-ph-api.herokuapp.com/?format=json'
+            setdata1(await axios.get(fetchUrl));
+            return 'ok';
+        }
+        fetchData();
+    },[]);
+    return data1
+}
+
+const GetWeatherData:FunctionComponent<{
```

Comment:
bakit pala 2 ?

Reply:
pinag araralan ko pa po kung anung best approach to get request  using ts language 
sa nakikita ko maganda yung use effect and axios  module

nasanay lang po ko sa AJAX ng js hehehe salamat po